config/default.go

```diff
@@ -110,6 +110,10 @@ MaxTxLifetime = "3h"
 		ByteGasCost = 16
 		MarginFactor = 1
 		Enabled = false
+	[Sequencer.StreamServer]
```

Comment:
StreamServer or DataStreamer? As we have used DataStreamerCfg for the config struct